### ISSUE TITLE
Typing in GPS coordinates causes the map to jump in real time

### ISSUE THREAD

**Issue by joshmc82** [iam4x/pokemongo-webspoof#159]
It would be nice if the location didn't try to update real time when you type in lat and lon coordinates directly. Ideally, I would like to be able to enter some coordinates, then hit "go" and get the option to Walk/Subway/Teleport.

Is that something that can be added in a future release?

**Comment by cheeseonhead** [comment]
Turn off auto update xcode location while typing and turn it back on after typing will do for now

**Comment by joshmc82** [comment]
Cool, that's a reasonable enough trick. I'll give it a shot.

**Comment by gonzaloh92** [comment]
@joshmc82 is it work for you?

**Comment by joshmc82** [comment]
Yep, the map on the Mac jumps but it.only updates on my phone when I check
the auto update box again.

**Comment by gonzaloh92** [comment]
@joshmc82 where do you type lat and lon coordinates? In app or Xcode? Do you get the option Walk/Subway? That would be nice to avoid soft ban.

**Comment by joshmc82** [comment]
You type it in the app on the Mac just under the search bar.
It is an instant teleport so you don't get the speed option unfortunately.
That was kinda the enhancement I was hoping for as well.

**Comment by TCB13** [comment]
> I would like to be able to enter some coordinates, then hit "go" and get the option to Walk/Subway/Teleport.

I would like to see this feature too! Thanks.
